kymatio/scattering1d/frontend/torch_frontend.py

```diff
@@ -27,27 +28,41 @@ def register_filters(self):
         saves those arrays as module's buffers."""
         n = 0
         # prepare for pytorch
+        # TODO: have option for different shaped filter depending on
+        # convolution primitive?
         for k in self.phi_f.keys():
             if type(k) != str:
                 # view(-1, 1).repeat(1, 2) because real numbers!
-                self.phi_f[k] = torch.from_numpy(
-                    self.phi_f[k]).float().view(-1, 1)
+                if self.conv_prim == 'spatial':
```

Comment:
Can you explain this?

Reply:
what is `spatial` -- i didn't see it elsewhere

Reply:
`spatial` is where we use the torch `Conv1d` module.

Reply:
I had it in, but then removed it. I guess I should remove this code then?

Reply:
OH. I guess we need to find a better name for that